test/api.js

```diff
@@ -314,9 +314,14 @@ test('test file in node_modules is ignored', function (t) {
 test('Node.js-style --require CLI argument', function (t) {
 	t.plan(1);
 
+	var requirePath = path.relative(process.cwd(), path.join(__dirname, 'fixture', 'install-global.js'));
+	// `path` will use \\ to seperate paths on windows, but require doesn't.
+	requirePath = requirePath.split(path.sep).join('/');
```

Comment:
I am not suggesting we change the implementation. Just the line here in the test that normalizes the path to posix. We don't need to do that. And we want the test to fail if it doesn't allow windows users to use it. `resolveCwd` is going to end up normalizing it for the given operating system anyways.